arangodb-net-standard/CollectionApi/CollectionApiClient.cs

```diff
@@ -144,11 +144,27 @@ public virtual async Task<GetCollectionsResponse> GetCollectionsAsync(GetCollect
         }
 
         /// <summary>
-        /// Gets the requested collection.
+        /// Return information about the requested collection.
         /// GET/_api/collection/{collection-name}
         /// </summary>
-        /// <param name="collectionName"></param>
-        /// <returns></returns>
+        /// <param name="collectionName">The name of the collection.</param>
+        /// <returns>
+        /// The result is an object describing the collection with the following attributes:
```

Comment:
Can you add related information to each properties in `GetCollectionResponse`? Like you have done for the `Type` property, this is very useful.

Here we can maybe keep the `returns` message shorter? e.g. "The result is an object describing the collection.". Easier to maintain, wouldn't you agree?